packages/astro/src/@types/astro.ts

```diff
@@ -2191,135 +2191,72 @@ export interface AstroUserConfig {
 		 * @name experimental.contentLayer
 		 * @type {boolean}
 		 * @default `false`
-		 * @version 4.16.0
+		 * @version 4.14.0
 		 * @description
 		 *
-		 * The Content Layer API is a new way to handle content and data in Astro. It takes [content collections](https://docs.astro.build/en/guides/content-collections/) beyond local files in `src/content` and allowing you to fetch content from anywhere, including remote APIs, or files anywhere in your project. As well as being more powerful, the Content Layer API is designed to be more performant, helping sites scale to tens of thousands of pages. Data is cached between builds and updated incrementally. Markdown parsing is also 5-10x faster, with similar scale reductions in memory. While the feature is experimental and subject to breaking changes, we invite you to try it today and let us know how it works for you.
+		 * The Content Layer API is a new way to handle content and data in Astro. It is similar to and builds upon [content collections](/en/guides/content-collections/), taking them beyond local files in `src/content/` and allowing you to fetch content from anywhere, including remote APIs, or files anywhere in your project by adding a `loader` to your collection.
```

Comment:
```suggestion
		 * The Content Layer API is a new way to handle content and data in Astro. It is similar to and builds upon [content collections](/en/guides/content-collections/), taking them beyond local files in `src/content/` and allowing you to fetch content from anywhere, including remote APIs, by adding a `loader` to your collection.
```
I'm wondering if this just sounds crisper and cleaner. "anywhere" really does cover it, but drawing extra attention to remote makes sense.